### PR TITLE
Bybit Heartbeat

### PR DESCRIPTION
**Pull request by jnatherley**
Fix issue with connection not being kept alive

### CODE REVIEW

File: src/exchange/bybit/message.rs [hunk 1]
```diff
@@ -1,11 +1,41 @@
+use crate::event::MarketIter;
```
Comment (just-a-stream) [inline src/exchange/bybit/message.rs:1]: `cargo +nightly fmt`

File: src/exchange/bybit/message.rs [hunk 2]
```diff
@@ -1,11 +1,41 @@
...
+///```json
+/// {
+///     "success": true,
+///     "ret_msg": "pong",
+///     "conn_id": "0970e817-426e-429a-a679-ff7f55e0b16a",
+///     "op": "ping"
+/// }
+/// ```
+#[derive(Clone, PartialEq, PartialOrd, Debug, Serialize, Deserialize)]
+pub struct BybitHeartbeat {
```
Comment (just-a-stream) [inline src/exchange/bybit/message.rs:32]: Do we need all of these fields? Perhaps was can just keep the `success` one?

File: src/exchange/bybit/message.rs [hunk 3]
```diff
@@ -67,8 +97,74 @@ where
...
+            BybitMessage::Trade(trade) => Some(trade.subscription_id.clone()),
+            _ => None,
+        }
+    }
+}
+
+impl From<(ExchangeId, Instrument, BybitMessage)> for MarketIter<PublicTrade> {
+    fn from((exchange_id, instrument, message): (ExchangeId, Instrument, BybitMessage)) -> Self {
+        match message {
+            BybitMessage::Heartbeat(_m) => Self(vec![]),
```
Comment (just-a-stream) [inline src/exchange/bybit/message.rs:112]:
```suggestion
            BybitMessage::Heartbeat(_) => Self(vec![]),
```

File: src/exchange/bybit/mod.rs [hunk 4]
```diff
@@ -29,7 +30,7 @@ pub mod futures;
 /// into an exchange [`Connector`] specific market used for generating [`Connector::requests`].
 pub mod market;
 
-/// Generic [`BybitMessage<T>`](message::BybitMessage) type common to
+/// Generic [`BybitMessage<T>`](message::BybitPayload) type common to
```
Comment (just-a-stream) [inline src/exchange/bybit/mod.rs:33]:
```suggestion
/// Generic [`BybitPayload<T>`](message::BybitPayload) type common to
```

File: src/exchange/bybit/message.rs (hunk 2 above)
Comment (jnatherley) [reply src/exchange/bybit/message.rs]: Think we can just encapsulate this all in the BybitResponse, ret_msg might not be necessary.

**Review by just-a-stream — APPROVED** [review]
LGTM

**Pull request merged**